datadog_checks_base/datadog_checks/base/data/agent_requirements.in

```diff
@@ -48,8 +48,7 @@ mmh3==2.5.1; python_version < '3.0'
 mmh3==3.0.0; python_version > '3.0'
 oauthlib==3.1.0; python_version < '3.0'
 oauthlib==3.2.2; python_version > '3.0'
-openstacksdk==0.39.0; python_version < '3.0'
-openstacksdk==0.61.0; python_version > '3.0'
+openstacksdk==1.0.1; python_version > '3.0'
```

Comment:
`keystoneauth1` should also be dropped

Reply:
I was doing it right now. Next commit.
Thanks for the head up!